File: 백준/Bronze/2675. 문자열 반복/문자열 반복.py
n = int(input())
for i in range(n):
    inp = input().split(' ')
    r, s = int(inp[0]), inp[1]
    arr = []
    for j in range(len(s)):
        for k in range(r):
            arr.append(s[j])
    print("".join(arr))

File: 백준/Bronze/2675. 문자열 반복/문자열 반복.cc
#include<stdio.h>

int main() {
  char arr[20];
  int t, r;
  scanf("%d", &t);
  for (int i = 0; i < t; i++) {
    scanf("%d %s", &r, arr);
    for (int j = 0; j < 20; j++) {
      if (arr[j] < 36 || arr[j] > 90) break;
      else if (arr[j] > 37 && arr[j] < 42) break;
      else if (arr[j] == 44) break;
      else if (arr[j] > 58 && arr[j] < 65) break;
      for (int k = 0; k < r; k++) {
        printf("%c", arr[j]);
      }
    }
    printf("\n");
  }
}